docs/content/en/docs/tasks/cluster/cluster-verify.md

```diff
@@ -19,26 +19,40 @@ capi-webhook-system                 capi-kubeadm-bootstrap-controller-manager-6b
 capi-webhook-system                 capi-kubeadm-control-plane-controller-manager-bf7878ffc-rgsm8    2/2     Running   0          47m
 capi-webhook-system                 capv-controller-manager-5668dbcd5-v5szb                          2/2     Running   0          47m
 capv-system                         capv-controller-manager-584886b7bd-f66hs                         2/2     Running   0          47m
-
 ```
 
-You may also check the status of the cluster control plane resource directly. 
-This can be especially useful to verify clusters with multiple control plane nodes after an upgrade.
-```
+**Example Output**
+
+You may also check the status of the cluster control plane resource directly, by running the following command:
+
+```bash
 kubectl get kubeadmcontrolplanes.controlplane.cluster.x-k8s.io
+```
+
+> Note: This can be especially useful to verify clusters with multiple control plane nodes after an upgrade.
+
+**Example Output**
+
+```bash
 NAME                       INITIALIZED   API SERVER AVAILABLE   VERSION              REPLICAS   READY   UPDATED   UNAVAILABLE
 supportbundletestcluster   true          true                   v1.20.7-eks-1-20-6   1          1       1
 ```
 
-To verify that the expected number of cluster worker nodes are up and running, use the `kubectl` command to show that nodes are `Ready`.
-This will confirm that the expected number of worker nodes, named following the format `$CLUSTERNAME-md-0`, are present.
-```
+* To verify that the expected number of cluster worker nodes are up and running, use the `kubectl` command to show that nodes are `Ready`.
+
+```bash
 kubectl get nodes
+```
+
+> Note: This will confirm that the expected number of worker nodes, named following the format `$CLUSTERNAME-md-0`, are present.
+
+**Example Output**
+
+```bash
```

Comment:
If commands are example output we don't include `bash` because the rendered site will add a copy button if bash is included. We don't want people copying the example output by mistake and trying to run it.